code/modules/research/rdconsole.dm

```diff
@@ -570,7 +548,8 @@ proc/CallMaterialName(ID)
 			compare = IMPRINTER
 			screen = 4.17

```

Comment:
Iirc this was used in legacy code to simulate tabbing or sub-category screen (pretty sure i shamelessly added to the mess when adding category grouping and search functionality to the lathes).
A strong candidate to tgui converting i guess.